test/FunctionalTests/Tests/DataServices/UnitTests/Client.TDD.Tests/Tests/RequestInfoTests.cs

```diff
@@ -134,11 +135,11 @@ public void RequestInfoShouldCreateTunneledPatchRequestMessagePostMethodAndPatch
             var descriptor = new EntityDescriptor(this.clientEdmModel) { ServerTypeName = this.serverTypeName, Entity = new Customer() };
             var buildingRequestArgs = new BuildingRequestEventArgs("PATCH", new Uri("http://localhost/fakeService.svc/"), headersCollection, descriptor, HttpStack.Auto);
 
-            var requestMessage = (HttpWebRequestMessage)testSubject.CreateRequestMessage(buildingRequestArgs);
+            var requestMessage = (HttpClientRequestMessage)testSubject.CreateRequestMessage(buildingRequestArgs);
 
             requestMessage.GetHeader(XmlConstants.HttpXMethod).Should().Be("PATCH");
             requestMessage.Method.Should().Be("PATCH");
-            requestMessage.HttpWebRequest.Method.Should().Be("POST");
+          //  requestMessage.HttpWebRequest.Method.Should().Be("POST");

```

Comment:
This probably needs to be completely removed, fixed, or replaced if no longer relevant